frontend/catalyst/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.2.0-dev"
+__version__ = "0.2.0"
```

Comment:
are there any other version strings where we will need to make version bumps?

Reply:
No. There are in fact two places that we use version strings: 1) `_version.py` 2) `changelog.md`